packages/contracts/test/exchange/core.ts

```diff
@@ -520,23 +532,39 @@ describe('Exchange core', () => {
         });
 
         it('should throw if maker allowances are too low to fill order', async () => {
-            await erc20TokenA.approve.sendTransactionAsync(erc20Proxy.address, new BigNumber(0), {
-                from: makerAddress,
-            });
-            expect(exchangeWrapper.fillOrderAsync(signedOrder, takerAddress)).to.be.rejectedWith(constants.REVERT);
-            await erc20TokenA.approve.sendTransactionAsync(erc20Proxy.address, constants.INITIAL_ERC20_ALLOWANCE, {
-                from: makerAddress,
-            });
+            await web3Wrapper.awaitTransactionMinedAsync(
+                await erc20TokenA.approve.sendTransactionAsync(erc20Proxy.address, new BigNumber(0), {
+                    from: makerAddress,
+                }),
+                constants.AWAIT_TRANSACTION_MINED_MS,
+            );
+            await expect(exchangeWrapper.fillOrderAsync(signedOrder, takerAddress)).to.be.rejectedWith(
```

Comment:
I think the reason we forgot to await these calls to `except` is because the `expect` method does not following our convention of suffixing async method names with `async`. 

Let's fix this by importing `expect` as `expectAsync`.

Reply:
Well, `expect` is not actually asynchronous. We often use it it synchronously in expressions like `expect(isAuthorized).to.be.false();`. It's the `to.be.rejectedWith` part that that's asynchronous and returns a promise.

I'm not sure if there's anything we can do here.